packages/server/graphql/public/typeDefs/Organization.graphql

```diff
@@ -137,6 +137,11 @@ type Organization {
   """
   orgUserCount: OrgUserCount!
 
+  """
+  Returns one on one team if exists
+  """
+  oneOnOneTeam(oneOnOneTeamInput: CreateOneOnOneTeamInput!): Team
```

Comment:
-1 this field should only get the `email` of the other user as input, as the org is already given by the parent